<ui version="4.0" >
 <class>PartEdit</class>
 <widget class="QWidget" name="PartEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="deviceGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="title" >
      <string>Partition 1 on FUJISTU - /dev/sda1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item rowspan="5" row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>12</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="label" >
        <property name="font" >
         <font/>
        </property>
        <property name="text" >
         <string>Device :</string>
        </property>
       </widget>
      </item>
      <item rowspan="5" row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>121</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="devicePath" >
        <property name="text" >
         <string>/dev/sda1</string>
        </property>
       </widget>
      </item>
      <item rowspan="4" row="0" column="6" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>90</width>
          <height>110</height>
         </size>
        </property>
       </spacer>
      </item>
      <item rowspan="5" row="0" column="7" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QPushButton" name="deletePartition" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet" >
           <string notr="true" />
          </property>
          <property name="text" >
           <string>Delete Partition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resizePartition" >
          <property name="text" >
           <string>Resize Partition</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="font" >
         <font/>
        </property>
        <property name="text" >
         <string>Use :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2" >
       <widget class="QComboBox" name="formatType" >
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text" >
          <string/>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>as Pardus System Files (mandatory)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>as User Files (optional)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>as Swap Space (optional)</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>as Storage Area</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font/>
        </property>
        <property name="text" >
         <string>Size :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QSlider" name="partitionSlider" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="4" >
       <widget class="QSpinBox" name="partitionSize" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="suffix" >
         <string> MB</string>
        </property>
        <property name="minimum" >
         <number>20</number>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="singleStep" >
         <number>50</number>
        </property>
        <property name="value" >
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font/>
        </property>
        <property name="text" >
         <string>Format :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="2" >
       <widget class="QCheckBox" name="formatCheck" >
        <property name="text" >
         <string>Yes, Format this partition</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" >
       <widget class="QLabel" name="label_7" >
        <property name="font" >
         <font/>
        </property>
        <property name="text" >
         <string>File System :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="fileSystem" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>29</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="fileSystemBox" >
          <item>
           <property name="text" >
            <string>Ext3</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Fat 32</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="5" >
       <widget class="QLabel" name="label_6" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>0</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="information" >
       <property name="styleSheet" >
        <string notr="true" />
       </property>
       <property name="text" >
        <string>Changes will be applied when you finish..</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetAllChanges" >
       <property name="text" >
        <string>Reset All Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyTheChanges" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>partitionSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>partitionSize</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>489</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>567</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>partitionSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>partitionSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>567</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel" >
     <x>489</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
